static/app/utils/analytics/growthAnalyticsEvents.tsx

```diff
@@ -105,8 +104,6 @@ type GrowthAnalyticsKey = keyof GrowthEventParameters;
 export const growthEventMap: Record<GrowthAnalyticsKey, string> = {
   'growth.show_mobile_prompt_banner': 'Growth: Show Mobile Prompt Banner',
```

Comment:
@Neo-Zhixing you forgot about this one too

Reply:
removed this and another one. this should be all